src/ign.in

```diff
@@ -31,10 +31,22 @@ required_tags =
 
 commands = {}
 yaml_found = false
+conf_dirs = []
+
+config_path = ENV['IGN_CONFIG_PATH'] if ENV.key?('IGN_CONFIG_PATH')
+if (config_path != nil)
+  config_path = config_path.split(":")
+  conf_dirs += config_path
+end
+conf_dirs.append('@CMAKE_INSTALL_PREFIX@')
+conf_dirs.append('sdformat')
+conf_dirs.append('ign_msgs')
+conf_dirs.append('ign_transport')
+conf_dirs.append('ign_transport/log') 
+conf_dirs.append('ign_fuel_tools')
+conf_dirs.append('ign_gui')
+conf_dirs.append('ign_gazebo')
```

Comment:
We shouldn't be hardcoding the libraries here, ideally they'd be registering themselves.